README.md

```diff
@@ -167,7 +167,7 @@ We will release a new major version from this library if a change in a supported
 
 ## Support
 
-This project, which integrates Drupal 8 with Apigee Edge, is supported by Google. Use our [project's issue queue](https://github.com/apigee/apigee-client-php/issues) to report any questions, issues, or feedback. 
+This project, which integrates Drupal 8 with Apigee Edge, is supported by Google. Use our [project's issue queue](https://github.com/apigee/apigee-client-php/issues) to report any questions, issues, or feedback.
```

Comment:
wdyt about "which connects" rather than "which integrates", since the php client really handles the communications while the integration is more at the module level?